TWLight/users/views.py

```diff
@@ -562,6 +562,12 @@ def delete(self, request, *args, **kwargs):
             user_authorization.date_expires = date.today() - timedelta(days=1)
             user_authorization.save()
 
+        # Delete bundle authorizations, if user is eligible for bundle
+        if user.editor.wp_bundle_authorized:
```

Comment:
As I understand it, this is a bit of a duplicate check - `wp_bundle_authorized` returns True if a bundle authorization exists (also using `get_bundle_authorization`), and then checks that the authorization `is_valid`. But in this context I don't think we actually care if the auth is valid or not (though it always should be), we'd want to delete it either way. So I think we can just jump straight to seeing if we get an object from `get_bundle_authorization` or not.

Reply:
I've changed the code to exclude that condition!